appveyor_after_build.ps1

```diff
@@ -224,21 +224,26 @@ function VerifyAnalysisResults
 {    
     WaitForAllBackgroundTasksToComplete
 
-    $expectedRuleViolations = @("fxcop:DoNotPassLiteralsAsLocalizedParameters", "fxcop:DoNotRaiseReservedExceptionTypes", "csharpsquid:S2228", "csharpsquid:S1134")
-
     $response = InvokeGetRestMethod "/api/issues/search?hideRules=true"
 
-    Assert ($response.paging.total -eq 4) "There should only be 4 issues"
-    Assert ($response.issues.Length -eq 4) "There should only be 4 issues"
```

Comment:
These checks cause the build to fail without producing any useful output to help debug the failure. I've removed these checks and replaced them with a single check to "response.total" below.